### PR TITLE
Implement RPC

### PR DESCRIPTION
**Pull request by iaj2**
- #9 Redone with RPC system. Can basically ignore any changes it's making to dev since it's a complete redo.

### CODE REVIEW

File: package.json
Comment (farzadpanahi) [inline package.json:1]: please update urls from osensa to edgepi-cloud.

**Review by farzadpanahi — APPROVED** [review]

File: edgepi-leds.html [hunk 1]
```diff
@@ -1,74 +1,89 @@
...
+                    ledPinRow.style.display = "none";
+                    receiveInputInfo.style.display = "block"
+                }
+            }
+            // Update and add event listener to update on user input
+            updateEditor()
+            configStyleSelect.addEventListener("change", updateEditor);
         }
     });
 </script>
```
Comment (naomiven) [inline edgepi-leds.html:40]: why is this javascript stuff in an html file? isn't it better to separate them?

**Review by naomiven — COMMENTED** [review]
looks good. just asked a question about having javascript in HTML file. i'm not sure what the best practice is for that, @iaj2 maybe you know more about it

File: edgepi-leds.html (hunk 1 above)
Comment (iaj2) [reply edgepi-leds.html:40]: Yes it's usually better to separate them. However in the case of node-red, this how they require it to be done. I'm not sure how it works under the hood on the node-red side, but has something to do with the way they parse the html file and generate the node UI.

File: package.json
Comment (iaj2) [reply package.json:1]: 162524e198633de309c4e3a46855344ed771b7bd

**Review by naomiven — APPROVED** [review]

**Review by farzadpanahi — APPROVED** [review]

**Pull request merged**